lib/apis/v3/felixconfig.go

```diff
@@ -180,6 +180,13 @@ type FelixConfigurationSpec struct {
 	VXLANPort *int `json:"vxlanPort,omitempty"`
 	VXLANVNI  *int `json:"vxlanVNI,omitempty"`
 
+	// AllowVXLANPacketsFromWorkloads controls whether Felix will add a rule to drop VXLAN encapsulated traffic
+	// from workloads [Default: false]
+	AllowVXLANPacketsFromWorkloads *bool `json:"allowVXLANPacketsFromWorkloads,omitempty"`
```

Comment:
Now that we're using Kubernetes CRD generation with schemas, I think we need to explicit about defining optional fields and defaults.

So, this field should have comments like this on it in order to generate the correct CRDs. 

```
// +optional
// kubebuilder:default:=false
```

Source: https://book.kubebuilder.io/reference/markers/crd-validation.html